app/views/shared/_order-tb.html.erb

```diff
@@ -0,0 +1,82 @@
+            <thead>
+                <tr>
+                    <th>#</th>
+                    <th>First Name</th>
+                    <th>Last Name</th>
+                    <th>Username</th>
+                    <th>User No.</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>1</td>
+                    <td><span class="label label-danger">Mark</span></td>
+                    <td>Otto</td>
+                    <td>@mdo</td>
+                    <td><span class="label label-info">100090</span></td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Jacob</td>
+                    <td>Thornton</td>
+                    <td>@fat</td>
+                    <td>100090</td>
+                </tr>
+                <tr>
+                    <td>3</td>
+                    <td>Larry</td>
+                    <td><span class="label label-danger">the Bird</span> </td>
+                    <td>@twitter</td>
+                    <td>100090</td>
+                </tr>
+                <tr>
+                    <td>4</td>
+                    <td><span class="label label-success">Mark</span></td>
+                    <td>Otto</td>
+                    <td>@mdo</td>
+                    <td><span class="label label-info">100090</span></td>
+                </tr>
+
+                <tr>
+                    <td>5</td>
+                    <td>Larry</td>
+                    <td><span class="label label-primary">the Bird</span></td>
+                    <td>@twitter</td>
+                    <td>100090</td>
+                </tr>
+                <tr>
+                    <td>6</td>
+                    <td><span class="label label-warning">Jacob</span></td>
+                    <td><span class="label label-success">Thornton</span></td>
+                    <td>@fat</td>
+                    <td><span class="label label-danger">100090</span></td>
+                </tr>
+                <tr>
+                    <td>7</td>
+                    <td>Larry</td>
+                    <td><span class="label label-primary">the Bird</span></td>
+                    <td>@twitter</td>
+                    <td>100090</td>
+                </tr>
+                <tr>
+                    <td>8</td>
+                    <td><span class="label label-warning">Jacob</span></td>
+                    <td><span class="label label-success">Thornton</span></td>
+                    <td>@fat</td>
+                    <td><span class="label label-danger">100090</span></td>
+                </tr>
+                <tr>
+                    <td>9</td>
+                    <td><span class="label label-success">Mark</span></td>
+                    <td>Otto</td>
+                    <td>@mdo</td>
+                    <td><span class="label label-info">100090</span></td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>
```

Comment:
add new line here as well.
Same here. we can add couple of entries rather then all these rows.